be/src/exec/olap_scan_node.h

```diff
@@ -266,6 +266,9 @@ class OlapScanNode : public ScanNode {
     RuntimeProfile::Counter* _block_fetch_timer = nullptr;
 
     RuntimeProfile::Counter* _index_load_timer = nullptr;
+
```

Comment:
Plz add comments to explain what these counters for~

Reply:
ok, I will add.